docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -0,0 +1,76 @@
+---
+id: getting-started-orchestrate-human-tasks
+title: Getting started with Human Task Orchestration
+sidebar_label: Getting started with Human Task Orchestration
+description: "Efficiently allocate work through user tasks."
+keywords: [human tasks, orchestration, getting started, user guide]
+---
+
+## Getting started with user task orchestration
+
+### Purpose
```

Comment:
You can go ahead and remove these two headers. The title is redundant with the frontmatter title value so it's duplicating and the purpose section doesn't need a title, just have it flow right into the text.

Reply:
Resolved in latest commit!